mailer.py

```diff
@@ -37,30 +38,33 @@ def send_message(*,
         return
 
     port = smtp_config.get(SMTP_PORT,  SMTP_PORT_DEFAULT)
-    debug = smtp_config.get(SMTP_DEBUG, SMTP_DEBUG_DEFAULT)
+    debug = smtp_config.get(SMTP_DEBUG, SMTP_DEBUG_DEFAULT)[0] in 'yYtT1'
```

Comment:
This line is failing with 

Traceback (most recent call last):
  File "/Users/sbarber/git/Plant-Tracer/webapp/.venv/lib/python3.11/site-packages/bottle.py", line 876, in _handle
    return route.call(**args)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/sbarber/git/Plant-Tracer/webapp/.venv/lib/python3.11/site-packages/bottle.py", line 1759, in wrapper
    rv = callback(*a, **ka)
         ^^^^^^^^^^^^^^^^^^
  File "/Users/sbarber/git/Plant-Tracer/webapp/bottle_app.py", line 345, in api_send_link
    db.send_links(email=email, planttracer_endpoint=planttracer_endpoint)
  File "/Users/sbarber/git/Plant-Tracer/webapp/db.py", line 142, in wrapper
    r = func(*args, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sbarber/git/Plant-Tracer/webapp/db.py", line 301, in send_links
    mailer.send_message(from_addr=PROJECT_EMAIL,
  File "/Users/sbarber/git/Plant-Tracer/webapp/mailer.py", line 41, in send_message
    debug = smtp_config.get(SMTP_DEBUG, SMTP_DEBUG_DEFAULT)[0] in 'yYtT1'
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
TypeError: 'bool' object is not subscriptable

That's because, I think, db.py 298 sets SMTP_DEBUG = False.

Reply:
And it wants to be a string